maven_iuvs/fits_processing.py

```diff
@@ -0,0 +1,498 @@
+import os as _os
+import datetime
+import numpy as np
+from pathlib import Path
+
+from maven_iuvs.search import find_files, get_latest_files
+
+
+def find_files_missing_geometry(file_index, show_total=False):
+    """
+    Identifies observation files with geometry
+
+    Parameters
+    ----------
+    file_index : index file (.npy) 
```

Comment:
I think the loaded index file is an array, not an .npy filename, so this might be confusing... I guess we could define a new datatype, but let's not if we can get away with it.